Machine-Learning/README.md

```diff
@@ -0,0 +1,106 @@
+- [Python in One Short -FreeCodeCamp(Youtube)](https://youtu.be/eWRfhZUzrAch)
+- [Python for absolute Begineer  -Corey Schafer(Full playlist including frameworks  and libraries)](https://youtube.com/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU)
+- [Intermidiate python in 6 hours -FreeCodeCamp](https://youtu.be/HGOBQPFzWKo)
+- [Python Tutorial -Tutorials point](https://www.tutorialspoint.com/python/index.html)
+- [Data Structures and algorithms in python -FreeCodeCamp](https://youtu.be/pkYVOmU3MgA)
+-[Robust Python: Write Clean and Maintainable Code by patrick viafore-pdf](https://dokumen.pub/download/robust-python-write-clean-and-maintainable-code-1nbsped-1098100662-9781098100667-c-7156903.html)
+
+## Get Familiar with Mathematics and Statistics
+
+### Linear Algebra
+- [Linear Algebra Full Course -FreeCodeCamp(Youtube)](https://youtu.be/JnTa9XtvmfI)
+- [Essence of linear algebra -3blue1brown](https://youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)
+- [Standford online -CS229(Youtube Playlist)](https://youtube.com/playlist?list=PLoROMvodv4rNH7qL6-efu_q2_bPuy0adh)
+- [Linear Algebra -MIT opencourseware](https://www.youtube.com/watch?v=JnTa9XtvmfI&t=0s)
+- [Introduction to Linear Algebra by David C. Lay -Book](https://home.cs.colorado.edu/~alko5368/lecturesCSCI2820/mathbook.pdf)
+- [10 Examples of Linear Algebra in Machine Learning -blog by Jason Brownlee](https://machinelearningmastery.com/examples-of-linear-algebra-in-machine-learning/)
+
+### Statistics and Probablity
+- [What is Statistics (and why is it important in machine learning)?](https://machinelearningmastery.com/what-is-statistics/)
+- [Statistics and Probablity -Edureka](https://youtu.be/XcLO4f1i4Yo)
+- [Statistics by FreeCodeCamp(Youtube)](https://youtu.be/xxpc-HPKN28)
+- [Probablity - Harvard University(Youtube)](https://youtube.com/playlist?list=PL2SOU6wwxB0uwwH80KTQ6ht66KWxbzTIo)
+- [Probablity-Khan Academy(Youtube)](https://youtube.com/playlist?list=PLC58778F28211FA19)
+- [Probablity for Machine Learning - Tyler Watkins](https://machinelearningmastery.com/probability-for-machine-learning/)
+- [Best Books for studying probablity for Machine Learning](https://www.javatpoint.com/probability-and-statistics-books-for-machine-learning)
+
+## Exploring Data Analysis and Visualization
+- [Data analysis with Python -FreeCodeCamp](https://youtu.be/GPVsHOlRBBI)
+- [Python Numpy Tutorial -FreeCodeCamp](https://youtu.be/QUT1VHiLmmI)
+- [Metaplotlib crash course -FreeCodeCamp](https://youtu.be/3Xc3CA655Y4)
+- [Python Pandas Tutorial -Corey Schafer](https://youtube.com/playlist?list=PL-osiE80TeTsWmV9i9c58mdDCSskIFdDS)
+- [Python Metaplotlib Tutorial -Corey Schafer](https://youtube.com/playlist?list=PL-osiE80TeTvipOqomVEeZ1HRrcEvtZB_)
+- [Pandas Documnetation](https://pandas.pydata.org/docs/)
+- [Numpy Documentation](https://numpy.org/doc/)
+- [Data analysis with Python -Tutorialspoint](https://www.tutorialspoint.com/data-analysis-and-visualization-in-python)
```

Comment:
Many errors in the resource. Beginners will have a hard time figuring them out.

Reply:
Please suggest me some other resources which I can add to make it more beginner-friendly

Reply:
Remove these resources as you say there are more similar resources already present in this document.